pkg/provision/primitives/zdb.go

```diff
@@ -141,7 +141,9 @@ func (p *Provisioner) createZdbContainer(ctx context.Context, allocation pkg.All
 	rootFS, err := flist.Mount(zdbFlistURL, "", pkg.MountOptions{
 		Limit:    10,
 		ReadOnly: false,
+		Type:     pkg.HDDDevice,
```

Comment:
Do you remember why we didn't set this to readonly ? Feel like this is the way to go so we don't have to allocate the subvolume for the 0-db other then the one for the actual data.

Reply:
I remember we had issues when the rootfs of zdb was readonly, but that was all what I remember. BTW this line is the actual cause of the issue.

Reply:
Then I think we need to have more logic. Cause now if there is no HDD, then we cannot deploy 0-db at all.
So I think we should try to deploy on any type of disks

Reply:
I agree, but this is not related to this issue. We need to create a new issue (also to retry read-only rootfs) but this now cause minio setup on devent to fail.